Updates_Docs

mariapiamo opened the pull request:
- Added in _Governance_process.pdf_ some examples of Governance-sensitive parameters that cannot be modified by the CM
- Updated the veOLAS createLockFor part in _Specifications of governance contracts_v1.0.1.pdf_  to better reflect the implementation

mariapiamo commented:
Done, I do not find links that refer to those documents to update though. Have a look when you can. In the _Governance_process.pdf_ the last sentence of the last page is different.

DavidMinarsch commented:
> Done, I do not find links that refer to those documents to update though. Have a look when you can. In the _Governance_process.pdf_ the last sentence of the last page is different.

Please can we reference the files in main readme then

mariapiamo commented:
In the README I added a line with the link to the contracts' specifications doc and I updated the link with the governance overview ( the second was already there as you said)
